CI is broken: ValueError: Either a revision or a version must be specified

CI is broken: https://github.com/huggingface/trl/actions/runs/26626672207/job/78465007507?pr=5877
> ValueError: Either a revision or a version must be specified

```python
=========================== short test summary info ============================
ERROR tests/distributed/test_distributed.py - ValueError: Either a revision or a version must be specified.
ERROR tests/distributed/test_distributed.py - ValueError: Either a revision or a version must be specified.
ERROR tests/distributed/test_distributed.py - ValueError: Either a revision or a version must be specified.
ERROR tests/distributed/test_distributed.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_activation_offloading.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_activation_offloading.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_activation_offloading.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_activation_offloading.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_callbacks.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_callbacks.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_callbacks.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_callbacks.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_chat_template_utils.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_chat_template_utils.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_chat_template_utils.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_chat_template_utils.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_cli.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_cli.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_cli.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_cli.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_cli_utils.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_cli_utils.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_cli_utils.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_cli_utils.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_data_utils.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_data_utils.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_data_utils.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_data_utils.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_dpo_trainer.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_dpo_trainer.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_dpo_trainer.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_dpo_trainer.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_grpo_trainer.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_grpo_trainer.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_grpo_trainer.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_grpo_trainer.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_model_utils.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_model_utils.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_model_utils.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_model_utils.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_reward_trainer.py - RuntimeError: Failed to import trl.trainer.reward_config because of the following error (look up to see its traceback):
Either a revision or a version must be specified.
ERROR tests/test_reward_trainer.py - RuntimeError: Failed to import trl.trainer.reward_config because of the following error (look up to see its traceback):
Either a revision or a version must be specified.
ERROR tests/test_reward_trainer.py - RuntimeError: Failed to import trl.trainer.reward_config because of the following error (look up to see its traceback):
Either a revision or a version must be specified.
ERROR tests/test_reward_trainer.py - RuntimeError: Failed to import trl.trainer.reward_config because of the following error (look up to see its traceback):
Either a revision or a version must be specified.
ERROR tests/test_rewards.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_rewards.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_rewards.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_rewards.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_rich_progress_callback.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_rich_progress_callback.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_rich_progress_callback.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_rich_progress_callback.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_rloo_trainer.py - RuntimeError: Failed to import trl.trainer.rloo_config because of the following error (look up to see its traceback):
Either a revision or a version must be specified.
ERROR tests/test_rloo_trainer.py - RuntimeError: Failed to import trl.trainer.rloo_config because of the following error (look up to see its traceback):
Either a revision or a version must be specified.
ERROR tests/test_rloo_trainer.py - RuntimeError: Failed to import trl.trainer.rloo_config because of the following error (look up to see its traceback):
Either a revision or a version must be specified.
ERROR tests/test_rloo_trainer.py - RuntimeError: Failed to import trl.trainer.rloo_config because of the following error (look up to see its traceback):
Either a revision or a version must be specified.
ERROR tests/test_sft_trainer.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_sft_trainer.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_sft_trainer.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_sft_trainer.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_utils.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_utils.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_utils.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_utils.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_vllm_client_server.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_vllm_client_server.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_vllm_client_server.py - ValueError: Either a revision or a version must be specified.
ERROR tests/test_vllm_client_server.py - ValueError: Either a revision or a version must be specified.
======================== 73 passed, 68 errors in 32.95s ========================
```

Stacktrace:
```python

______________ ERROR collecting tests/test_vllm_client_server.py _______________
tests/test_vllm_client_server.py:21: in <module>
    from transformers import AutoModelForCausalLM, AutoProcessor, AutoTokenizer
<frozen importlib._bootstrap>:1075: in _handle_fromlist
    ???
.venv/lib/python3.10/site-packages/transformers/utils/import_utils.py:2249: in __getattr__
    module = self._get_module(self._class_to_module[name])
.venv/lib/python3.10/site-packages/transformers/utils/import_utils.py:2483: in _get_module
    raise e
.venv/lib/python3.10/site-packages/transformers/utils/import_utils.py:2481: in _get_module
    return importlib.import_module("." + module_name, self.__name__)
/__w/_tool/Python/3.10.20/x64/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
.venv/lib/python3.10/site-packages/transformers/models/auto/processing_auto.py:25: in <module>
    from ...image_processing_utils import ImageProcessingMixin
.venv/lib/python3.10/site-packages/transformers/image_processing_utils.py:34: in <module>
    from .processing_utils import ImagesKwargs, Unpack
.venv/lib/python3.10/site-packages/transformers/processing_utils.py:63: in <module>
    from .utils.type_validators import (
.venv/lib/python3.10/site-packages/transformers/utils/type_validators.py:19: in <module>
    from ..activations import ACT2FN
.venv/lib/python3.10/site-packages/transformers/activations.py:22: in <module>
    from .integrations.hub_kernels import use_kernel_forward_from_hub
.venv/lib/python3.10/site-packages/transformers/integrations/hub_kernels.py:89: in <module>
    "cuda": LayerRepository(
.venv/lib/python3.10/site-packages/kernels/layer/layer.py:76: in __init__
    raise ValueError("Either a revision or a version must be specified.")
E   ValueError: Either a revision or a version must be specified.
```

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -53,7 +53,7 @@ deepspeed = [
     "transformers!=5.1.0",  # see transformers#43780
 ]
 kernels = [
-    "kernels"
+    "kernels<0.15.1",  # see GH-5878
 ]
 liger = [
     "liger-kernel>=0.8.0"
@@ -104,7 +104,7 @@ dev = [
     # deepspeed
     "deepspeed>=0.14.4",
     # kernels
-    "kernels",
+    "kernels<0.15.1",  # see GH-5878
     # liger
     "liger-kernel>=0.8.0",
     # openreward (requires Python 3.11+)

